apps/web-giddh/src/app/shared/header/header.component.ts

```diff
@@ -294,10 +295,18 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
         private settingsProfileService: SettingsProfileService,
         private settingsProfileAction: SettingsProfileActions,
         private companyService: CompanyService,
+        private _breakPointObservar: BreakpointObserver,
```

Comment:
don't use _ undescore in private objects/var
```suggestion
        private breakPointObservar: BreakpointObserver,
```